core/src/main/java/org/infinispan/functional/EntryView.java

```diff
@@ -118,6 +119,88 @@ private EntryView() {
     * @since 8.0
     */
    @Experimental
-   public interface ReadWriteEntryView<K, V> extends ReadEntryView<K, V>, WriteEntryView<V> {}
+   public interface ReadWriteEntryView<K, V> extends ReadEntryView<K, V>, WriteEntryView<V> {
+      /**
+       * Sets this value to the result of <code>updateFunction</code> applied
+       * to the current value, or <code>null</code> if absent.
+       * If <code>updateFunction</code> returns <code>null</code>, the mapping
+       * is removed.
+       *
+       * @param updateFunction
+       * @return <code>null</code>
+       * @since 9.1
+       */
+      default Void update(Function<? super V, ? extends V> updateFunction) {
+         return set(updateFunction.apply(find().orElse(null)));
+      }
+
+      /**
+       * Sets this value to the result of <code>computeFunction</code>
+       * applied to the current value, or <code>null</code> if absent.
+       * If <code>computeFunction</code> returns <code>null</code>, the mapping
+       * is removed.
+       *
+       * @param computeFunction
+       * @return Value returned by the <code>computeFunction</code>
+       * @since 9.1
+       */
+      default V compute(Function<? super V, ? extends V> computeFunction) {
+         V newValue = computeFunction.apply(find().orElse(null));
+         set(newValue);
+         return newValue;
+      }
+
+      /**
+       * Set this value to value provided by the <code>supplier</code> if there
+       * is no current mapping.
+       *
+       * @param supplier
+       * @return <code>null</code>
+       * @since 9.1
+       */
+      default Void setIfAbsent(Supplier<? extends V> supplier) {
```

Comment:
`set` takes an immediate value, so `setIfAbsent` should do the same (not that I think a method like that would be very useful).